### PR TITLE
fix alertmanager-overview dashboard angular deprecated dashboards

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR migrates the old angular panels to react (used the grafana migrate button)

Old

image

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Comment by hervenicol** [comment]
I guess that's the panels you graphs from the dashboard, not the whole dashboard :sweat_smile: 
 
Why removing them?
They couldn't be fixed? Or they were useless?

**Comment by QuentinBisson** [comment]
🤦🏻 This PR migrates the old angular dashboards, let me fix the description

**Comment by QuentinBisson** [comment]
Now, I'm not sure we really use this dashboard, but that's besides the point

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add missing data source to atlas dashboards.
 - Fix missing provider specific dashboards.
+- Upgrade old angular plugins in atlas dashboards.
```
Comment (hervenicol) [inline CHANGELOG.md:18]: This would make more sense IMO:
```suggestion
- Udate alertmanager-overview dashboard (angular deprecation).
```

Comment (QuentinBisson) [reply CHANGELOG.md:18]: It would but I upgraded a bunch of others that I forgot to mention here. Let me fix it

Comment (QuentinBisson) [reply CHANGELOG.md:18]:
```suggestion
- Update alertmanager-overview dashboard (angular deprecation).
- Update fluentbit dashboard (angular deprecation).
- Update prometheus-overview dashboard (angular deprecation).
- Update prometheus-remote-write dashboard (angular deprecation).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add missing data source to atlas dashboards.
 - Fix missing provider specific dashboards.
+- Update alertmanager-overview dashboard (angular deprecation).
+- Update fluentbit dashboard (angular deprecation).
+- Update operatorkit dashboard (angular deprecation).
```
Comment (QuentinBisson) [inline CHANGELOG.md:20]: Adding changelog for https://github.com/giantswarm/dashboards/pull/558/files and https://github.com/giantswarm/dashboards/pull/557 as well

**Review by hervenicol — APPROVED** [review]
:+1:

**Pull request merged**